pyperf/_process_time.py

```diff
@@ -26,9 +26,11 @@
     resource = None
 
 
-def get_max_rss():
+def get_max_rss(resource_type=None):
     if resource is not None:
-        usage = resource.getrusage(resource.RUSAGE_CHILDREN)
+        if resource_type is None:
+            resource_type = resource.RUSAGE_CHILDREN
```

Comment:
We should not use RUSAGE_SELF on Linux, FreeBSD and other platforms?

Reply:
This function is called in two contexts -- one when the benchmark runs entirely in a separate subprocess, in which case we want `CHILDREN`, and one when in which it runs in the same process, where we want `SELF`.  Saying that, it would probably clearer to not have a default value for `resource_type` here.